example_notebooks/example_files/CP_files/mitocheck_idr_cp.cppipe

```diff
@@ -28,7 +28,7 @@ Metadata:[module_num:2|svn_version:'Unknown'|variable_revision_number:6|show_win
     Does cached metadata exist?:No
     Metadata extraction method:Import from file
     Metadata source:File name
-    Regular expression to extract from file name:(?P<Plate>LT[0-9]{4}_[0-9][0-9])_(?P<Well_Number>[0-9]{1,3})_(?P<Frames>50)
+    Regular expression to extract from file name:(?P<Plate>LT[0-9]{4}_[0-9][0-9])_(?P<Well_Number>[0-9]{1,3})_(?P<Frames>[0-9]{1,3})
```

Comment:
No, it looks like we both missed the second line where a regular expression with frames is used and only changed the first line. Below is a screenshot of the current version of [mitocheck_idr_cp.cppipe](https://github.com/WayScience/IDR_stream/blob/main/example_notebooks/example_files/CP_files/mitocheck_idr_cp.cppipe):
![image](https://user-images.githubusercontent.com/105665675/221245394-152a3776-19b3-408d-b67a-77f602af2fdd.png)

(Notice the very bottom line is still using the wrong regular expression for `Frames`)

Also, I missed the change to `?P<Plate>LT[0-9]{4}_[0-9]{2}` in this bottom line, which has been added in https://github.com/WayScience/IDR_stream/pull/21/commits/95cac866f2c8157207a41efac81d1aad1f8a1363

Reply:
Ahh good find! This is interesting that it shows this line twice and differently. This is an example of why I only change things within the GUI since I know for sure I changed it in the right parameter. Not sure why there are two lines, but this should still work!